### DISCUSSION TITLE
java image into github pages

### DISCUSSION THREAD

**Discussion by BenWill151** [community/community#35636] [New to GitHub]
I was wondering how to get an image into my code for GitHub since I can just put it in the sketch files and the 
img = loadImage("d20-blank.png"); will just draw from there in processing. I was wondering where I can put the image so that the code I run in GitHub pages can pull the image.

first image is it working with image in processing, second image is it working but not showing the dice behind the numbers on github pages.
![c60a22d1bbf829e74c661886182ca5f3](https://user-images.githubusercontent.com/111608888/194798911-1869f518-5c1f-4d14-8750-688c17ff14a6.png)
![abb082bff295d7be392ff7d208f92580](https://user-images.githubusercontent.com/111608888/194798942-8282a5d3-a072-40d9-9bc5-87c1ff31c7f0.png)

**Comment by jfullstackdev** [comment]
you mean embedding an image in html ? to be displayed on your GitHub Page site ? like the one you presented above ? 

it depends whether you are using standard HTML or Markdown

for HTML, check this out 

https://www.w3schools.com/html/html_images.asp

for Markdown, particularly here at GitHub, simply copy the image and paste it, you will see the raw Markdown like

```
![image](https://user-images.githubusercontent.com/47092464/194879723-9719708f-8391-4a1a-b009-155a20cc0e6d.png)
```

when you render it, it's the image you copied

**Reply by BenWill151** [reply]
void show()
  {
    img = loadImage("d20-blank.png");
    R1 = (int)(Math.random()*19)+1;
    img.resize(150,150);
    image(img, myX, myY);
    fill(255,255,255);
    textSize(30);
    text(R1, myX-2,myY);
  }
}

in my java code it calls the image and i dont know where to put that image so the dice.pde program will grab it

**Reply by jfullstackdev** [reply]
how is your Java code rendered to GitHub Pages ? because GitHub Pages only accepts plain HTML, CSS and JavaScript or Markdown,

you might want to find another external service that will render your Java code to a website and the rendered code for that will be the one you will be uploading on GitHub
